pyanaconda/ui/gui/spokes/advstorage/dasd.py

```diff
@@ -76,6 +78,10 @@ def run(self):
             try_populate_devicetree(self.storage.devicetree)
         return rc
 
+    def _set_configure_sensitive(self, sensitivity):
+        """ Set entries to a given sensitivity. """
+        self._deviceEntry.set_sensitive(sensitivity)
+
```

Comment:
I don't know but is it better to call
``self._set_configure_sensitive(True)``
instead of
``self._deviceEntry.set_sensitive(True)``
so much to create new function for it?

Or do you have a better reason why this function?